models/intermediate/field_history/int_jira__pivot_timestamp_field_history.sql

```diff
@@ -0,0 +1,99 @@
+with issue_field_history as (
+
+    select *
+    from {{ ref('int_jira__issue_field_history') }}
+
+),
+
+issue_multiselect_history as (
+
+    select *
+    from {{ ref('int_jira__issue_multiselect_history') }}
+),
+
+issue_multiselect_batch_history as (
+    -- Aggregate multiselect field values into comma-separated strings
+    select
+        field_id,
+        field_name,
+        issue_id,
+        updated_at,
+        author_id,
+        {{ fivetran_utils.string_agg('field_value', "', '") }} as field_values
+
+    from issue_multiselect_history
+    {{ dbt_utils.group_by(5) }}
+),
+
+combine_field_history as (
+    -- Union single-select and multiselect field histories
+    select
+        field_id,
+        issue_id,
+        updated_at,
+        author_id,
+        field_value,
+        field_name
+    from issue_field_history
+
+    union all
+
+    select
+        field_id,
+        issue_id,
+        updated_at,
+        author_id,
+        field_values as field_value,
+        field_name
+    from issue_multiselect_batch_history
+),
+
+limit_to_relevant_fields as (
+    -- Filter to only status and configured custom fields
+    select
+        combine_field_history.*
+    from combine_field_history
+    where lower(field_id) = 'status'
+        or lower(field_name) in ('sprint', 'story points', 'story point estimate'
+        {%- for col in var('issue_field_history_columns', []) -%}
+            ,'{{ (col|lower) }}'
+        {%- endfor -%} )
```

Comment:
Since this model isn't used to populate the downstream sprint_enhanced or sprint issue field history models, we can probably remove the story points and story point estimate fields here.